src/pages/get-started.page.js

```diff
@@ -11,12 +11,18 @@ import Container from '@material-ui/core/Container';
 import * as queryString from 'query-string';
 import SnackbarMessage from '@general/alerts/snackbar-message';
 
+import GAService from '@services/ga.service';
+
 export default class GetStartedPage extends Component {
   static contextType = AppContext;
   state = {};
 
   constructor() {
     super();
+
+    this.gaService = GAService.getInstance();
```

Comment:
i think we could probably refactor this later into a base page component that has this init stuff in there